client/src/components/Overview/AddToCart.jsx

```diff
@@ -150,31 +204,38 @@ const AddToCart = () => {
   };
 
   // ADD TO CART BUTTON ========================================================
+  const postToCartOnce = (skuId) => {
+    axios.post('/cart', { sku_id: skuId })
+      .then((response) => response.status)
+      .catch((err) => console.log('Error posting to cart', err));
+  };
+
   const handleClick = () => {
     // if clicked without selecting a size, show error message and open size dropdown
     if (selectSize === 'Select Size') {
       showError(true);
       showSizes(true);
     } else {
       const item = {
-        sku: selectSku[0],
-        quantity: selectQty,
-        size: selectSize,
+        sku_id: selectSku[0],
+        count: selectQty,
       };
-      addToCart([...cart, item]);
+      for (let i = 0; i < selectQty; i += 1) {
+        postToCartOnce(selectSku[0]);
+      }
       resetDefault();
+      console.log('Added to cart:', item);
```

Comment:
Can you remove these console logs on line 227 and 228

Reply:
@kyang9021 Thanks for reviewing. I've removed the console.logs on line 227 and 228. 

I also tested the add to cart feature again and it worked on my end. The number of items I selected were removed from the quantity dropdown. The cart shown in the console with a console.log won't reflect the addition because the cart is rendered via an async get request which takes time to fetch the data. Showing the user cart isn't part of the requirements so rendering the correct number isn't necessary.